sdcm/db_stats.py

```diff
@@ -120,9 +120,13 @@ def get_stress_cmd_params(cmd):
                 if 'throttle' in cmd_params['rate']:
                     cmd_params['throttle threads'] =\
                         re.search(r'(throttle\s?=\s?(\w+))', cmd_params['rate']).group(2)
-                if 'fixed' in cmd_params['rate']:
+                if 'fixed=' in cmd_params['rate']:
                     cmd_params['fixed threads'] =\
                         re.search(r'(fixed\s?=\s?(\w+))', cmd_params['rate']).group(2)
+                elif 'fixed' in cmd_params['rate']:
+                    cmd_params['fixed threads'] =\
+                        re.search(r'(throttle\s?=\s?(\w+))', cmd_params['rate']).group(2)
+                    del cmd_params['throttle threads']
```

Comment:
remove this change after merging https://github.com/scylladb/cql-stress/pull/127 and releasing